quiche/src/lib.rs

```diff
@@ -2734,6 +2734,9 @@ impl Connection {
     ///  * When the application receives data from the peer (for example any
     ///    time [`stream_recv()`] is called).
     ///
+    /// Once [`is_draining`] returns `true`, it is no longer necessary to call
+    /// `send()` and all calls will return [`Done`].
```

Comment:
Let's hyper link it here and elsewhere. E.g. take the suggestion here and then in the trailing section with other hyperlinks add `//! [`send()`]: struct.Connection.html#method.send`

```suggestion
    /// [`send()`] and all calls will return [`Done`].
```